controllers/homePageAPI.js

```diff
@@ -1,17 +1,20 @@
 const express = require('express');
-const homePage = require('../models/homePage')
-const view = require('../views/homePageView');
 const router = express.Router();
 
 router.get('/', (req, res) =>
 {
 	console.log("Receieved req for homePage listings");
-	homePage.GetRecentListings(
```

Comment:
why did you delete this? we need to display the listings provied (from the database) on the home page, currently we are just going to show the title (in text), in the future we will just use a picture.